src/controllers/zone_metadata.cr

```diff
@@ -0,0 +1,122 @@
+require "promise"
+
+require "./application"
+
+module PlaceOS::Api
+  class ZoneMetadata < Application
+    include Utils::CoreHelper
+    base "/api/engine/v2/zones/:id"
```

Comment:
Can you add a comment that this is mounted under the zones path?

Reply:
done